lib/tasks/assignments.rake

```diff
@@ -3,6 +3,33 @@ namespace :db do
   desc 'Create assignments'
   task :assignments => :environment do
     # Assignments
+
+    rule = GracePeriodSubmissionRule.new
+    assignment_stat = AssignmentStat.new
+    puts 'Assignment 0: Grace Token usage'
+    Assignment.create(
+      short_identifier: 'A0',
+      description: 'Variables and Simple Operations',
+      message: 'using basic operators and assigning variables',
+      group_min: 1,
+      group_max: 1,
+      student_form_groups: true,
+      group_name_autogenerated: true,
+      group_name_displayed: false,
+      repository_folder: 'A0',
+      due_date: 12.hour.ago,
```

Comment:
I believe ruby allows you to use both singular and plural forms for dates and times; please try `12.hours.ago` and see how it goes. (Similar comment below.)